README.md

```diff
@@ -1,9 +1,11 @@
-# Northcoders News API
+# Northcoders News API - Back-End - ME
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+## Instructions on how to connect to two databases
 
+1. Please make two files in the root directory named 
+**.env.test**  and **.env.development**
 
+2. In the **.env.test** file please enter database name :  **PGDATABASE=nc_news_test**
 
---- 
+3. In the **.env.development** file please enter database name :  **PGDATABASE=nc_news**
```

Comment:
This is fine, but it might be more correct to let the user choose his or her database name - it doesn't necessarily *need* to be called nc_news and nc_news_test :)